### PR TITLE
Extend parameters supplied to build analysis

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/messaging/pull/330

## This introduces a breaking change

- [x] Yes

### CODE REVIEW

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1121,33 +1121,87 @@ def schedule_dependency_monkey(
...
+            base_image_analysis_id = self.generate_id("package-extract")
+
+        if output_image and not output_image_analysis_id:
+            output_image_analysis_id = self.generate_id("package-extract")
+
+        if buildlog_document_id and not buildlog_parser_id:
+            buildlog_parser_id = self.generate_id("buildlog-parser")
+
+        if not job_id:
+            job_id = self.generate_id("build-analysis")
```
Comment (pacospace) [inline thoth/common/openshift.py:1182]: should be buildlogs-analysis? https://github.com/fridex/storages/blob/33f9b76217f17595603f83118658854c871b8651/thoth/storages/buildlogs_analyses.py#L26

**Review by pacospace — COMMENTED** [review]
One comment, otherwise lgtm!

File: thoth/common/openshift.py (hunk 1 above)
Comment (fridex) [reply thoth/common/openshift.py:1182]: I wanted to keep this more generic - it's not just build log analysis, but analysis of the whole build - including resulting container images.

Comment (pacospace) [reply thoth/common/openshift.py:1182]: Thanks for explanation :), then all good for me!

**Review by pacospace — APPROVED** [review]
:100:

**Pull request merged**